skbio/stats/ordination/tests/data/PCoA_sample_data_3_fsvd_3d_ref

```diff
@@ -0,0 +1,22 @@
+Eigvals	3
+0.26252020959180306	0.09043197375300857	0.07177687511060699
+
+Proportion explained	3
+0.6180886481996106	0.2129168512318794	0.16899450056851006
```

Comment:
Are these proportions reasonably concordant with PCoA using `eigh`?

Reply:
...it looks like they sum to 1 here which means PC1 variation will likely change substantially if 3 dimensions are requested vs. 10, right?

Reply:
fsvd to 3 dimensions:
eigh produces the following proportions on the same dataset, which all sum to 1:
`0.47397595644762963	0.2781864326674703	0.24783761088490003`

Reply:
Is this with or without the squaring?

Reply:
With trace and after I fixed erroneous squaring of eigvals:

eigh
```
Eigvals	3
0.5123672604605051	0.30071909442702155	0.26791206600414047
```

fsvd
```
Eigvals	3
0.5123672604605054	0.3007190944270222	0.2679120660041405
```